cache/edge_lambdas/toggler.js

```diff
@@ -10,65 +10,92 @@
 // }
 
 // This is mutable for testing
-let tests = [];
+let tests = [
+  {
+    id: 'useBetaDownloadComponent',
```

Comment:
Happy to be calling things `beta` and `new`, but it's not massively descriptive if someone sees the toggles without seeing the PR.

Perhaps we could go for really far out code names?
https://github.com/technosophos/moniker